tests/test_plot.py

```diff
@@ -187,40 +187,49 @@ def test_2d_axes_limits():
                 assert(axes[z][y].get_ylim() == (c, d))
 
 
-def test_kde_plot_1d():
+@pytest.mark.parametrize('plot_1d', [kde_plot_1d, fastkde_plot_1d])
+def test_kde_plot_1d(plot_1d):
```

Comment:
@williamjameshandley, did you know about the `pytest.mark.parametrize` functionality? Learnt about it recently. Super useful. That way there is no need for for-loops and when a test fails, it tells you which item caused the issue.

Reply:
I have seen it used in the past, but never managed to find the time to implement it in anesthetic. I agree that this would make the whole test suite far neater. Do you want to raise an issue/PR that starts this off?